app/lib/report_helper.rb

```diff
@@ -50,11 +55,25 @@ def taxonomy_details(view_level, report, params)
       )
     end
     sort_report!(nt, params[:sort_by])
+
     nr_zscore_by_taxon = nr.group_by { |h| h[:tax_id] }
+    genus_nr_zscore_by_taxon = nr_genus.group_by { |h| h[:tax_id] }
+
     tax_details = []
     nt.each do |h|
```

Comment:
This takes all the NT hits and displays them with their corresponding NR scores, if any. It does not show hits that were only present in NR but not in NT. However, those hits should also be displayed. Could you add that in, by refactoring so that and nt and nr are treated equivalently?